### ISSUE TITLE
Make make_md.py immune to spaces in BibTeX file fields

### ISSUE THREAD

**Issue by matthewfeickert** [iml-wg/HEPML-LivingReview#43]
At the moment [`make_md.py`](https://github.com/iml-wg/HEPML-LivingReview/blob/99a8b087e737d2e296537b736822a03096ad7702/make_md.py) is sensitive to the space between after the `=` in `title =` and the start of the title field itself.

For example, looking at the entry 

https://github.com/iml-wg/HEPML-LivingReview/blob/99a8b087e737d2e296537b736822a03096ad7702/HEPML.bib#L2567-L2578

as is

https://github.com/iml-wg/HEPML-LivingReview/blob/99a8b087e737d2e296537b736822a03096ad7702/HEPML.bib#L2575

`make_md.py` produces

```markdown
        * [ Jet Substructure at the Large Hadron Collider: A Review of Recent Advances in Theory and Machine Learning](https://arxiv.org/abs/1709.04464) [[DOI](https://doi.org/10.1016/j.physrep.2019.11.001)]
```

however

```
title ="{Jet Substructure at the Large Hadron Collider: A Review of Recent Advances in Theory and Machine Learning}", 
```

produces

```markdown
        * [Jet Substructure at the Large Hadron Collider: A Review of Recent Advances in Theory and Machine Learning](https://arxiv.org/abs/1709.04464) [[DOI](https://doi.org/10.1016/j.physrep.2019.11.001)]
```

While Markdown should render this the same, variations like this should be avoided to reduce the amount of unnecessary diffs that can come from autoformatting. So we should teach `make_md.py` to strip out the `title` field entry itself.

**Comment by bnachman** [comment]
Fixed.

**Comment by matthewfeickert** [comment]
Closed by 9175a64
